### ISSUE TITLE
PronType=Rel vs. Int

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#278]
It appears there are instances of `PronType=Int` that should be `Rel`, assuming that `PronType=Rel` is correct for any relativizer in the presence of `acl:relcl`.

For example, here are WDT determiners within a relative clause: http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=61bffccb7016f

See also #88

**Comment by nschneid** [comment]
Ack maybe these are not actually relativizers. They are just interrogative NPs extracted from the RC.

"offset whatever gains could be made"—"that" can be inserted I think: "offset whatever gains that could be made"

**Comment by amir-zeldes** [comment]
I think this one is a WH-determiner - whatever gains is like "what gains", or "those gains" in the non-interrogative context.

**Comment by nschneid** [comment]
Right. Even though the whole sentence isn't interrogative, I think [`PronType=Int`](https://universaldependencies.org/en/feat/PronType.html#int-interrogative-pronoun-determiner-or-adverb) is correct for non-relativizer WDTs:

<img width="866" alt="image" src="https://user-images.githubusercontent.com/985263/146841911-497cf31e-d21c-4be4-bd60-809e4cd21540.png">

**Comment by amir-zeldes** [comment]
OK, I think that's what we have in GUM ATM too

**Comment by nschneid** [comment]
OK, these queries should work better—they reveal a bunch of `PronType=Int` or `Dem` that look like they should be `Rel`:

- [q0 (E:ref)](http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=61db5b78306cf&clustering=W.PronType)
- [q1 (1 hop)](http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=61db5b90229d5&clustering=W.PronType)
- [q2 (2 hops)](http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=61db5c25554c7&clustering=W.PronType)
- [q3 (3 hops)](http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=61db5c903a6e7&clustering=W.PronType)
- [q4 (4 hops)](http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=61db5cca8bdd8&clustering=W.PronType)

...and likewise for GUM.

Note that q1-q4 are needed due to errors in the `E:ref` annotations.

**Comment by nschneid** [comment]
Most of the missing E:ref edges that I'm having to add manually are due to cases where (1) multiple RCs share the same true head (coordinated RCs, successive RCs), or (2) the extraction is from a clause nested within the RC.

**Comment by nschneid** [comment]
- [ ] from q2: I'm undecided about the structure of "it will give you some idea of **what it might cost** and **what is fair**"

**Comment by amir-zeldes** [comment]
I think it's a coordinate free relative to an nmod modifying "idea" ("idea of that which it might cost/of that which is fair" - both WH phrases stand for NPs):

nmod(idea,what1)
acl:relcl(what1,cost)
conj(what1,what2)
acl:relcl(what2,fair)

**Comment by nschneid** [comment]
I discussed this with Brett Reynolds on Twitter and it convinced me it was interrogative. https://twitter.com/YanisLing/status/1485921746775683074

The difference between free relatives and interrogative content clauses (a.k.a. indirect questions) is extremely subtle and hard to tell just from the meaning/paraphrasing. But basically the idea is that if it's an environment that licenses "whether" clauses, it's probably interrogative.

**Comment by nschneid** [comment]
[ ] as to what the light provides it gives UV rays..**which** is important for health in lizards as it is in humans. 

Do we think the "which"-RC refers to the NP headed by "rays" (despite number agreement mismatch), or just "UV"? The current annotation is inconsistent between these two interpretations.

**Comment by amir-zeldes** [comment]
> But basically the idea is that if it's an environment that licenses "whether" clauses, it's probably interrogative.

No, sorry I don't get it: the sentence was give you some idea of what it might cost", there is no whether here. Of course you can say "give you some idea of whether it would cost something", but the same is true for prototypical free relative environments:

  * See what Kim brought
  * See whether Kim brought something

I agree the second is a content clause, but the first is a regular free relative, right? It means "see the thing which Kim brought" (which is an NP, and the sense can be split into the two roles, i.e. the "thing which" paraphrase)

> it gives UV rays..which is important

I think the agreement suggests that the relcl expands on the whole "gives" predication, i.e. the clause is a dependent of "gives". In GUM such cases can be found by looking for verbs dominating a relative clause directly:

https://corpling.uis.georgetown.edu/annis/#_q=dXBvcz0iVkVSQiIgLT5kZXAgdG9rX2Z1bmM9ImFjbDpyZWxjbCI&_c=R1VN&cl=5&cr=5&s=0&l=10&o=random

**Comment by nschneid** [comment]
Thanks, I'll adopt that analysis for UV rays.

Re: free relatives: This is tricky. I think "see" is ambiguous. Note that it can accept either a direct object ("I saw the book") or a complement clause ("I saw that Kim brought something").

If the sentence were

- I need to see what Kim brought and whether anyone else brought anything.

I would interpret that as coordination of two interrogative complement clauses. It is about trying to discover something, so it is like saying "I want to discover the answer to the questions, What did Kim bring? and Did anyone else bring anything?" (but when those questions are embedded their form changes).

A canonical example of a free relative is with a verb like *eat* that doesn't license complement clauses:

- I'll eat **what you cook me**.
  * \*I'll eat whether you cook the food.

At [this page](https://universaldependencies.org/en/dep/acl-relcl.html) I've documented some of the main syntactic tests, including *else* (which favors interrogatives) and *-ever* (which only works for free relatives):

- I'll eat **whatever** you cook me. \*I'll eat **what else** you cook me. => Free relative
- \*I need to see **whatever** Kim brought and whether anyone else brought anything. I need to see **what else** Kim brought and whether anyone else brought anything. => Interrogative

In isolation, "I need to see what Kim brought" is ambiguous. It can imply 'there was something that Kim brought, and I need to see it' => free relative, or it can indicate an investigation into what Kim brought ('there was something that Kim brought, and I need to see _what_') => interrogative. Which leads to yet another test: 

- There was something that Kim cooked, and I need to see/explain/*eat what.

**Comment by nschneid** [comment]
"see" also #122

**Comment by nschneid** [comment]
Stumbled upon another piece of evidence regarding the free relative/interrogative distinction: with expressions like *(not) give a damn* (and maybe also *have no idea* though my intuitions there are less sharp), what follows has to be a clause, not an NP, because then it would have 2 direct objects. Thus:

- I don't give a damn [what you cooked]/ccomp
- *I don't give a damn [whatever you cooked]. (*whatever*-clauses are free relatives)
- *I don't give a damn [your choice of food].

Likewise, "what you cooked" can be parsed as `ccomp` in "I don't care/know what you cooked" (*know* licenses a complement clause or direct object; *care* maybe as well: *?I don't care your reasons!*).

**Comment by nschneid** [comment]
- [ ] ["which" PronType=Int](http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=61f62e224aba2&clustering=rel.label) tokens need reviewing. A lot are missing an `acl:relcl` annotation.

**Comment by nschneid** [comment]
Looking at [the cases annotated as "what" free relatives with PronType=Int](http://universal.grew.fr/?custom=63e1cbb173868), I interpret many of these as interrogative clauses instead. Changing the ones headed by: bad, certain, discuss, find, give, moment, remember, restaurant, reveal, say, see, tell.

(“explain”, “hear”, “understand” could license interrogatives but in these cases I think the free relative interpretation is better)

**Comment by nschneid** [comment]
Tricky one: "She...listens to **what it is you would like to achieve**". Sort of a mix of a pseudocleft and an it-cleft! We can treat it as such:

obj(listens, what)
acl:relcl(what, is)
expl(is, it)
advcl:relcl(what, like)
xcomp(like, achieve)

E:obj(achieve, what)

**Comment by nschneid** [comment]
[matrix-subject "what" free relatives with PronType=Int](http://universal.grew.fr/?custom=63e8579ca8d68)

**Comment by nschneid** [comment]
[matrix-copular "what" free relatives with PronType=Int](http://universal.grew.fr/?custom=63e86ae6107f1)

**Comment by nschneid** [comment]
- [ ] [q6 but check for missing edeprel from embedded clause](http://universal.grew.fr/?custom=643c577a8c036)

**Comment by nschneid** [comment]
- [ ] [q7: WH-word is `det` or `advmod`](http://universal.grew.fr/?custom=643c60408191f)
- [x] [q8: WH-word is "whose": double-check that this is `PronType=Rel`](http://universal.grew.fr/?custom=643c611d22a18)

**Comment by nschneid** [comment]
- [ ] [q9: csubj clauses](https://universal.grew.fr/?custom=6736271f78519) from the above-linked thread
